packages/@sanity/default-layout/src/main/NotFound.tsx

```diff
@@ -2,39 +2,38 @@
 ///<reference types="@sanity/types/parts" />
 
 import React from 'react'
-import {withRouterHOC, StateLink} from 'part:@sanity/base/router'
+import {StateLink} from '@sanity/state-router/components'
+import {withRouterHOC} from 'part:@sanity/base/router'
+import {Card, Text, Box, Stack, Container, Heading} from '@sanity/ui'
 import {HAS_SPACES} from '../util/spaces'
 import {Router} from '../types'
 
-import styles from './NotFound.css'
-
-interface OuterProps {
-  children: React.ReactNode
-}
-
 interface NotFoundProps {
   children: React.ReactNode
   router: Router
 }
 
-function NotFound(props: NotFoundProps) {
+export const NotFound = withRouterHOC((props: NotFoundProps) => {
```

Comment:
This HOC can be replaced by `import {useRouterState} from '@sanity/base/router'`